resources/console/profile-evaluation.yaml

```diff
@@ -0,0 +1,14 @@
+web:
+  deployment:
+    resources:
+      requests:
+        memory: 16Mi
```

Comment:
On two different clusters I checked, the pod from this deployment consumes exactly 19Mi (2Mi Console; 17Mi Core-ui) while running. I think there's no case when it'd take less, so I suggest to use this value as the `request`.

Reply:
Also, here's the `weekly` cluster 12h summary. A bit different figures but still, I think it should be much less for the `console` container (as you currently apply) or much more for the whole pod.
![image](https://user-images.githubusercontent.com/22825433/100615736-500a0e00-3318-11eb-87bf-0db10e6ba486.png)

Reply:
This is the evaluation profile, so weekly is not a good source of information. This cluster runs lots of tests all the time, and we won't have those tests in the evaluation profile.